### PR TITLE
First version

### PR DESCRIPTION
**Pull request by wal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+Add to config/deploy.rb:
+
+```ruby
+    require 'kitman-cap-autoscale'
+```
+
+Add to config/deploy/(staging | production).rb:
+
+```ruby
+set :autoscaling_group_name, 'medinah-production'
```
Comment (rod-murphy) [inline README.md]: Should we change 'medinah-production' to something more generic?

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,61 @@
+PATH
```
Comment (chrismcg) [inline Gemfile.lock]: You shouldn't checkin a Gemfile.lock for a gem, just for applications.

**Comment by rod-murphy** [comment]
LGTM

File: Gemfile.lock (hunk 2 above)
Comment (chrismcg) [inline Gemfile.lock]: http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/ ^^ ref for above.

**Comment by chrismcg** [comment]
:shipit:

**Pull request merged**